### ISSUE TITLE
Where is the atom loss and bond loss?

### ISSUE THREAD

**Issue by windyLemon** [zaixizhang/MGSSL#8]
Dear author, in the paper the final loss is the sum of atom, bond and motif. I cannot find the loss of atom and bond in the source code. So what is the final loss function of MGSSL?

**Comment by zaixizhang** [comment]
For the conciseness of the code, we here only upload the motif-based SSL task codes in the GitHub repo. The atom/bond loss code can be found in related works such as https://github.com/snap-stanford/pretrain-gnns/
